### PR TITLE
refactor: add more precise verbalization about checkboxes

### PR DESCRIPTION
**Pull request by elukashova**
- [x] change verbalization about checkboxes to avoid misunderstanding about their quantity

### CODE REVIEW

File: react/modules/module02/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Task should be based on the previous task.
      text input - name, surname (or both), zip-code;
      date input - birthday, date of delivery;
      dropdown/select - list of countries, list of states (User can choose only one element from the list)
-     checkbox - "I consent to my personal data" field, list of extra presents (User can choose several items from the list)
+     checkbox - both one and multiple checkboxes are accepted: "I consent to my personal data" field, list of extra presents (User can choose several items from the list)
```
Comment (ragingyngvarr) [inline react/modules/module02/README.md:27]: What is the reason for this change?

Comment (elukashova) [reply react/modules/module02/README.md:27]: some discussions in the chat today
